# Overall state of the boat
Header header
bool enabled
bool autonomous
bool presetOne
